Description: let Qt fall back to C++11/GCC atomics for MIPS
 The "previous" value is always 1 when the compare-and-swap succeeded,
 instead of the previous value.
 .
 Instead of fixing this, let's just remove this file a bit earlier than
 the rest. All of them will be removed in Qt 5.7 anyway, so let's leave
 MIPS atomics to the compiler.
Origin: upstream, http://code.qt.io/cgit/qt/qtbase.git/commit/?id=110a8c339fa078a4
Bug: https://bugreports.qt.io/browse/QTBUG-49168
Last-Update: 2015-11-16

--- a/src/corelib/thread/qbasicatomic.h
+++ b/src/corelib/thread/qbasicatomic.h
@@ -60,8 +60,6 @@
 # include "QtCore/qatomic_armv5.h"
 #elif defined(Q_PROCESSOR_IA64)
 #  include "QtCore/qatomic_ia64.h"
-#elif defined(Q_PROCESSOR_MIPS)
-#  include "QtCore/qatomic_mips.h"
 #elif defined(Q_PROCESSOR_X86)
 #  include <QtCore/qatomic_x86.h>
 
